### PR TITLE
conf-openblas: fix for opensuse

### PR DESCRIPTION
**Pull request by mor1**

### CODE REVIEW

File: packages/conf-openblas/conf-openblas.0.2.0/opam [hunk 1]
```diff
@@ -6,17 +6,14 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
...
+  ["sh" "-exc" "cc $CFLAGS test.c -lopenblas"] {os != "fedora" & os != "centos" & os != "opensuse" & os != "darwin"}
 ]
 depexts: [
-  [["fedora"] ["openblas-devel"]]
-  [["centos"] ["epel-release" "openblas-devel"]]
   [["alpine"] ["libc-dev" "openblas-dev"]]
-  [["ubuntu"] ["libopenblas-dev" "liblapacke-dev"]]
-  [["debian"] ["libopenblas-dev" "liblapacke-dev"]]
+  [["centos"] ["epel-release" "openblas-devel"]]
+  [["debian" "ubuntu"] ["libopenblas-dev" "liblapacke-dev"]]
```
Comment (kit-ty-kate) [inline packages/conf-openblas/conf-openblas.0.2.0/opam]: this is not the expected syntax here. The first element expects a list of flags that are going to be there on the user's machine. In this case the machine cannot be both `debian` and `ubuntu`.
These should be separated as they were before.

File: packages/conf-openblas/conf-openblas.0.2.0/opam [hunk 2]
```diff
@@ -6,17 +6,14 @@ bug-reports: "https://github.com/ocaml/opam-repository/issues"
 dev-repo: "https://github.com/ocaml/opam-repository.git"
 license: "BSD"
 build: [
-  ["sh" "-exc" "cc $CFLAGS test.c -lopenblas"] {os = "alpine"}
-  ["sh" "-exc" "cc $CFLAGS test.c -lopenblas"] {os = "debian"}
```
Comment (kit-ty-kate) [inline packages/conf-openblas/conf-openblas.0.2.0/opam]: `debian` has been removed. This shouldn't be the case I suppose

Comment (mor1) [reply packages/conf-openblas/conf-openblas.0.2.0/opam]: This case should still be covered by the following line (`{os != "fedora" ...}`) I think?

File: packages/conf-openblas/conf-openblas.0.2.0/opam (hunk 1 above)
Comment (mor1) [reply packages/conf-openblas/conf-openblas.0.2.0/opam]: Ah apologies, misinterpreted the format. (Couldn't find docs specifying it :(

Comment (kit-ty-kate) [reply packages/conf-openblas/conf-openblas.0.2.0/opam]: here is the documentation: https://opam.ocaml.org/doc/Manual.html

File: packages/conf-openblas/conf-openblas.0.2.0/opam (hunk 2 above)
Comment (kit-ty-kate) [reply packages/conf-openblas/conf-openblas.0.2.0/opam]: Yes, you are right. Sorry.

**Comment by kit-ty-kate** [comment]
Looks ok now.

**Comment by kit-ty-kate** [comment]
Thanks.

**Comment by mor1** [comment]
Ah thanks! Could've sworn I looked at that before, must've missed the
relevant bit...

**Comment by kit-ty-kate** [comment]
@mor1 no worries. It happens. The CIs are here to check that.

**Pull request merged**